src/main/java/ro/developmentfactory/thedrones/service/DronePositionService.java

```diff
@@ -0,0 +1,12 @@
+package ro.developmentfactory.thedrones.service;
+
+import ro.developmentfactory.thedrones.controller.dto.DroneStatusResponse;
+
+public interface DronePositionService {
+
+    DroneStatusResponse turnDroneLeft(DroneStatusResponse droneStatusResponse);
+
+    DroneStatusResponse turnDroneRight(DroneStatusResponse droneStatusResponse);
```

Comment:
if you have `POST /drones/{id}/turn/L` + `POST /drones/{id}/turn/R` you'll have to use only 1 method for turning and pass ex: `turningDirection` (L or R) to the method